portality/static/js/doaj.js

```diff
@@ -0,0 +1,126 @@
+            }
+        }
+
+        var toc_id = undefined;
+        if (eissns.length > 0) { toc_id = eissns[0]; }
+        if (!toc_id && pissns.length > 0) { toc_id = pissns[0]; }
+        if (!toc_id) { toc_id = journal.id; }
+
+        return toc_id;
+    },
+
+    monthmap : [
+        "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sept", "Oct", "Nov", "Dec"
+    ],
+
+    licenceMap : {
+        "CC BY" : ["/static/doaj/images/cc/by.png", "https://creativecommons.org/licenses/by/4.0/"],
+        "CC BY-NC" : ["/static/doaj/images/cc/by-nc.png", "https://creativecommons.org/licenses/by-nc/4.0/"],
+        "CC BY-NC-ND" : ["/static/doaj/images/cc/by-nc-nd.png", "https://creativecommons.org/licenses/by-nc-nd/4.0/"],
+        "CC BY-NC-SA" : ["/static/doaj/images/cc/by-nc-sa.png", "https://creativecommons.org/licenses/by-nc-sa/4.0/"],
+        "CC BY-ND" : ["/static/doaj/images/cc/by-nd.png", "https://creativecommons.org/licenses/by-nd/4.0/"],
+        "CC BY-SA" : ["/static/doaj/images/cc/by-sa.png", "https://creativecommons.org/licenses/by-sa/4.0/"]
+    },
+
+    humanDate : function(datestr) {
+        var date = new Date(datestr);
+        var dom = date.getUTCDate();
+        var monthnum = date.getUTCMonth();
+        var year = date.getUTCFullYear();
+
+        return String(dom) + " " + doaj.monthmap[monthnum] + " " + String(year);
+    },
+
+    humanDateTime : function(datestr) {
+        var date = new Date(datestr);
+        var dom = date.getUTCDate();
+        var monthnum = date.getUTCMonth();
+        var year = date.getUTCFullYear();
+        var hour = date.getUTCHours();
+        var minute = date.getUTCMinutes();
+
+        if (String(hour).length === 1) {
+            hour = "0" + String(hour);
+        }
+
+        if (String(minute).length === 1) {
+            minute = "0" + String(minute);
+        }
+
+        return String(dom) + " " + doaj.monthmap[monthnum] + " " + String(year) + " at " + String(hour) + ":" + String(minute);
+    },
+
+    iso_datetime2date : function(isodate_str) {
+        /* >>> '2003-04-03T00:00:00Z'.substring(0,10)
+         * "2003-04-03"
+         */
+        return isodate_str.substring(0,10);
+    },
+
+    iso_datetime2date_and_time : function(isodate_str) {
+        /* >>> "2013-12-13T22:35:42Z".replace('T',' ').replace('Z','')
+         * "2013-12-13 22:35:42"
+         */
+        if (!isodate_str) { return "" }
+        return isodate_str.replace('T',' ').replace('Z','')
+    }
+};
+
+jQuery(document).ready(function() {
+    $.noop(); // just a placeholder, delete when adding code here
+});
+
+
+function setCookieConsent(event) {
+    event.preventDefault();
+    $.ajax({
+        type: "GET",
+        url: "/cookie_consent",
+        success: function() {
+            $("#cookie-consent-banner").remove();
+        },
+        error : function() {
+            alert("We weren't able to set your cookie consent preferences, please try again later.");
+        }
+    })
+}
```

Comment:
I think for the moment I'm not going to do this.  The reason is that everything else on the site is ES5 compatible, and doing this would make this one file ES6. I've so far held off on moving to ES6 features, because of the various old browsers that we want to support.  I'm not against looking to upgrade, but I don't think this PR is the place to do it.